spec/features/participants/participant_tracker/identity_changes_participant_status_spec.rb

```diff
@@ -28,4 +28,10 @@ def when_i_update_the_participant_status
   def then_i_should_see_the_updated_status
     expect(bootstrap_selected?("participant_status_#{@participant.id}", "Screening")).to be
   end
+
+  def and_i_should_see_an_associated_note
```

Comment:
You added this method but never implemented it into the test. Also, it should probably be placed in its own test and changed to a then... method.